vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditSpaceLinks.java

```diff
@@ -0,0 +1,50 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+public class EditSpaceLinks {
+
+    public ResponseEntity<String> checkIfSpaceExists(ISpaceManager spaceManager, String id, String x, String y) throws IOException{
+        ISpace source = spaceManager.getSpace(id);
+        if (source == null) {
+            return new ResponseEntity<>("{'error': 'Space could not be found.'}", HttpStatus.NOT_FOUND);
+        }
+        if (x == null || x.trim().isEmpty() || y == null || y.trim().isEmpty()) {
+            ObjectMapper mapper = new ObjectMapper();
+            ObjectNode node = mapper.createObjectNode();
+            node.put("errorMessage", "No link coordinates specified.");
+            return new ResponseEntity<String>(mapper.writeValueAsString(node), HttpStatus.BAD_REQUEST);
+        }
+        return null;
+    }
+    
+    public ResponseEntity<String> imageCouldNotBeStored() throws IOException{
```

Comment:
is that being used? also, it doesn't look like the image can be changed currently?

Reply:
Removed the block of code that is not used.
No, the image on space is not editable. Should I work on making image type and alert type links editable?

Reply:
all properties of links need to be editable